app/models/requirement_block.rb

```diff
@@ -4,6 +4,9 @@ class RequirementBlock < ApplicationRecord
   has_many :requirement_block_requirements, -> { order(position: :asc) }, dependent: :destroy
   has_many :requirements, through: :requirement_block_requirements
 
+  has_many :requirement_template_section_requirement_blocks, dependent: :destroy
+  has_many :requirement_templates, through: :requirement_template_requirement_blocks
```

Comment:
Cannot fin this relationship `:requirement_template_requirement_blocks`

Reply:
typo. will fix.  good catch.